Marlin/src/gcode/calibrate/M665.cpp

```diff
@@ -44,10 +44,7 @@
    *    Z = Rotate A and B by this angle
    */
   void GcodeSuite::M665() {
-    if (parser.seen('H')) {
-      delta_height = parser.value_linear_units();
-      update_software_endstops(Z_AXIS);
-    }
```

Comment:
Note that in `update_software_endstops` we have this:
```cpp
      soft_endstop_max[axis] = (axis == Z_AXIS ? delta_height : base_max_pos(axis));
```
If you skip the call, this line will never be invoked.

Reply:
It will because homing is required after a change. You were going to set up a 'sandbox project' in order to try to get rid of that requirement, but for as long as this is not functional, we'll have to stick to this.

Leaving this statement is misleading; it gives the false impression that it works without homing for delta_height changes.